File: src/test/java/org/jesperancinha/airports/containers/CountryContainerTest.java
package org.jesperancinha.airports.containers;

import org.jesperancinha.airports.pojos.Country;
import org.junit.Test;

import java.util.List;

import static org.hamcrest.Matchers.hasSize;
import static org.hamcrest.Matchers.notNullValue;
import static org.junit.Assert.assertThat;

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
public class CountryContainerTest {
    @Test
    public void getCountries() throws Exception {
        final CountryContainer airportContainer = new CountryContainer();

        List<Country> countries = airportContainer.getCountries();
        Country country = countries.get(0);

        assertThat(countries, hasSize(248));
        assertThat(country.getId(), notNullValue());
    }

}

File: src/test/kotlin/org/jesperancinha/airports/containers/CountryContainerTest.kt
package org.jesperancinha.airports.containers

import org.hamcrest.Matchers
import org.jesperancinha.airports.pojos.Country
import org.junit.Assert
import org.junit.Test

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
class CountryContainerTest {
    @Test
    fun `should run country test container`(){
            val airportContainer = CountryContainer()
            val countries: List<Country> = airportContainer.countries
            val country = countries[0]
            Assert.assertThat(countries, Matchers.hasSize(248))
            Assert.assertThat(country.id, Matchers.notNullValue())
        }
}